### PR TITLE
One file for apply and destroy

### PR DESCRIPTION
**Pull request by Koufan-De-King**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -82,9 +82,17 @@ jobs:
     if: ${{ inputs.apply-or-destroy != 'none' }}
     needs: [tflint, tfsec]
     runs-on: ubuntu-latest
+    name: Terraform ${{ matrix.name }} Environment
+    environment:
+      name: ${{ matrix.name }}
+      url: https://${{ vars.DOMAIN_NAME }}
```
Comment (stephane-segning) [inline .github/workflows/deploy.yaml:88]: Remove the url from this one, as we're destroying

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -82,9 +82,17 @@ jobs:
...
     strategy:
-      matrix:
-        name: [dev]
+      matrix: 
+        name:
+          - dev
+        include:
+          - name: dev
+            description: "Environment to be changed"
```
Comment (stephane-segning) [inline .github/workflows/deploy.yaml:95]: Do you need this include block?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Review by stephane-segning — CHANGES_REQUESTED** [review]
We don't want a single file for apply and destroy. We want them separated

**Comment by Koufan-De-King** [comment]
Okay @stephane-segning.
I thought it a wise idea for making destroy use the same sate as apply.
Will close the PR now.

**Pull request closed without merging**